thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/dimensions/template.hbs

```diff
@@ -3,6 +3,15 @@
     {{#link-to "manage.alert.explore" record.anomaly.functionId (query-params duration=record.queryDuration startDate=record.queryStart endDate=record.queryEnd) class="te-anomaly-table__link"}}
       {{record.anomaly.functionName}}
     {{/link-to}}
+
+    {{!--  TODO: move this into a link above each metric table - Report missing anomaly link
+    {{#link-to "manage.alert.explore" record.anomaly.functionId (query-params duration=record.queryDuration startDate=record.queryStart endDate=record.queryEnd openReport=true) class="te-anomaly-table__link te-anomaly-table__link--glyph" target="_blank"}}
+      <i class="glyphicon glyphicon-edit"></i>
+      {{#tooltip-on-element class="te-anomaly-table__tooltip"}}
+        Report missing anomaly for this alert.
+      {{/tooltip-on-element}}
+    {{/link-to}} --}}
```

Comment:
should this still be here?

Reply:
I just want it there for the next iteration - didn't want to sink time into it this time.